src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -28,12 +35,21 @@ const options: Partial<Options> = {
     placement: 'bottom-start',
 };
 
-const PopupList = ({ reference, ...rest }: Props): JSX.Element => (
-    <PopupBase className="ba-PopupList" options={options} reference={reference} {...rest}>
-        <div className="ba-PopupList-prompt ba-PopupList-item">
-            <FormattedMessage {...messages.popupListPrompt} />
-        </div>
-    </PopupBase>
-);
-
-export default PopupList;
+export default function PopupList<T extends { id: string }>({
+    items,
+    reference,
+    options,
+    ...rest
+}: Props<T>): JSX.Element {
+    return (
+        <PopupBase className="ba-PopupList" options={{ ...defaultOptions, ...options }} reference={reference}>
+            {isEmpty(items) ? (
+                <div className="ba-PopupList-prompt">
+                    <FormattedMessage {...messages.popupListPrompt} />
```

Comment:
Should there be a separate message for when a user has typed in a filter query but there are no results?

Reply:
> Should there be a separate message for when a user has typed in a filter query but there are no results?

Waiting for designer's response.

Reply:
Rohit said that is ok